Makefile

```diff
@@ -2026,6 +2029,19 @@ telemetry_deploy_cleanup: ## cleans up the service instance, Does not affect the
 	${CLEANUP_DIR_CMD} ${OPERATOR_BASE_DIR}/ceilometer-operator ${DEPLOY_DIR}
 	oc rsh -t $(DBSERVICE_CONTAINER) mysql -u root --password=${PASSWORD} -e "flush tables; drop database if exists aodh;" || true
 
+.PHONY: telemetry_kuttl_run
+telemetry_kuttl_run: ## runs kuttl tests for the telemetry operator, assumes that everything needed for running the test was deployed beforehand.
+	kubectl-kuttl test --config ${TELEMETRY_KUTTL_CONF} ${TELEMETRY_KUTTL_DIR} --namespace ${NAMESPACE}
+
+.PHONY: telemetry_kuttl
+telemetry_kuttl: export NAMESPACE = ${TELEMETRY_KUTTL_NAMESPACE}
+# Set the value of $TELEMETRY_KUTTL_NAMESPACE if you want to run the telemetry
+# kuttl tests in a namespace different than the default (telemetry-kuttl-tests)
+telemetry_kuttl: input deploy_cleanup telemetry telemetry_deploy_prep
+	make wait
```

Comment:
seems like it's missing ` $(eval $(call vars,$@,telemetry))` before this line to run correctly